src/Controller/User/Delete.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\User;
+
+use App\Entity\User;
+use OpenApi\Attributes as OA;
+use Doctrine\ORM\EntityManagerInterface;
+use Nelmio\ApiDocBundle\Annotation\Model;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class Delete extends AbstractController
+{
+    #[Route('/api/users/{uuid}', methods: ['DELETE'])]
+    #[OA\Response(
+        response: 204,
+        description: 'Returns the user\'s information after being deleted',
```

Comment:
https://www.abstractapi.com/http-status-codes/204#:~:text=The%20HTTP%20Status%20Code%20204,is%20not%20returning%20any%20content.